packages/itwinui-react/src/core/utils/components/InputContainer.tsx

```diff
@@ -52,10 +50,13 @@ export const InputContainer = <T extends React.ElementType = 'div'>(
       data-iui-status={status}
       data-iui-label-placement={isLabelInline ? 'inline' : undefined}
       style={style}
+      ref={forwardedRef}
       {...rest}
     >
       {label && (
         <Label
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore -- shut up for the love of god
```

Comment:
Maybe don't keep this comment.

Reply:
oh wow. lets pretend you didn't see that